testflo/summary.py

```diff
@@ -74,3 +75,12 @@ def get_iter(self, input_iter):
             procstr = " using %d processes" % self.options.num_procs
         write("\n\nRan %d test%s%s\nWall clock time:   %s\n\n" %
                       (total, s, procstr, elapsed_str(wallclock)))
+
+        if fails:
+            sys_info = platform.uname()
+            write("\nYour system's specifications are shown below "
+                  "to facilitate reporting of failures.")
+            write("\n\nOS:     %s\nCPU:    %s\nPython: %s\n\n" %
+                  ("%s | %s" % (sys_info.system, sys_info.version),
+                   "%s | %s" % (sys_info.machine, sys_info.processor),
+                   sys.version))
```

Comment:
Would you object to doing this only in the `testflo_report.out` file and not printing it to the screen?

When using testflo In day-to-day development, we wouldn't want to see this extraneous output all the time as we debug tests.  It seems more appropriate for inclusion in the report file that could then be submitted as you suggest.

If so, I think you would want to add a flag to the `ResultSummary` constructor and enable it in `main.py` only for the instance that writes to the report file.

Also..  this code is python 3 specific... we need to handle python 2.7 as well...

Reply:
TBH I had forgotten about this PR.  Not offended if you decline it.